webpack/assets/javascripts/react_app/components/HostDetails/Audits/index.js

```diff
@@ -1,143 +1,101 @@
-                              <DataListItemRow>
-                                <DataListItemCells
-                                  dataListCells={[
-                                    <DataListCell key={key}>
-                                      <span> {key}</span>
-                                    </DataListCell>,
-                                    <DataListCell
-                                      key={`old-value-${i}`}
-                                      style={{
-                                        width: '350px',
-                                        overflowX: 'hidden',
-                                      }}
-                                    >
-                                      {value && (
-                                        <ElipsisWithTooltip>
-                                          <mark
-                                            style={{
-                                              backgroundColor: '#FFC0CB',
-                                            }}
-                                          >
-                                            {value[0]}
-                                          </mark>
-                                        </ElipsisWithTooltip>
-                                      )}
-                                    </DataListCell>,
-                                    <DataListCell
-                                      style={{
-                                        width: '350px',
-                                        overflowX: 'hidden',
-                                      }}
-                                      key={`new-value-${i}`}
-                                    >
-                                      {value && (
-                                        <mark
-                                          style={{
-                                            backgroundColor: '#90EE90',
-                                          }}
-                                        >
-                                          {value[1]}
-                                        </mark>
-                                      )}
-                                    </DataListCell>,
-                                  ]}
-                                />
-                              </DataListItemRow>
-                            </DataListItem>
-                          )
-                        )}
-                    </DataList>
-                  </AccordionContent>
-                </AccordionItem>
-              );
-            return null;
-          })}
-        </Accordion>
+        <SkeletonLoader
+          skeletonProps={{ count: 3 }}
+          status={status}
+          emptyState={
+            <Bullseye>
+              <Title headingLevel="h4"> {__('No Results found')} </Title>
+            </Bullseye>
+          }
+        >
+          <DataList aria-label="recent-audit-table" isCompact>
```

Comment:
would this not prevent the skeleton loader from displaying the emptyState when there are no results? as it would have children